src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProvider.cs

```diff
@@ -92,6 +94,11 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
 
                 foreach (var member in members)
                 {
+                    if (IsInvalidCompletionMethodKind(member))
```

Comment:
i think this all can be removed.  We alreayd have an IsAccessor extension method you can use here :)

Reply:
https://github.com/dotnet/roslyn/blob/2d37f3ca6f8df83315ef7e2116e494c8574c9158/src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs#L822-L825

Reply:
Ah man, I feel embarrassed I didn't think about checking for an extension method. Thanks for pointing this out and stopping me from re-inventing the wheel.

Reply:
Hey, I forgot to check too. :smile: